src/style.css

```diff
@@ -1,3 +1,160 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:500,500italic);
+
+:root {
+  --accent-color: #00acdc;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+*,
+*:before,
+*:after {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
```

Comment:
1. Just `*` without `:before` and `:after`
2. Vendor prefixes are not needed too.